.ci/README.md

```diff
@@ -1,63 +1,87 @@
 
-After that, the generated repositories are uploaded to GitHub, in the concourse process runner's forks.  The pull request is updated to point to those forks.
+The Magician will take over, running the generator and running the downstream tests.  If the tests fail, you'll see "check failed" status on your PR<!--TODO(@ndmckinley) - better reporting and logs of test failures.-->.  When all the tests pass, the PR or PRs created by The Magician will link back to your MagicModules PR.  From there, it is your responsibility to engage with *all* the downstream PRs - The Magician doesn't test or merge those unless otherwise indicated.  You cannot merge your MagicModules PR until *all* downstreams accept the changes.  If a downstream requests changes and you need to modify your MagicModules PR, simply push another commit to your MagicModules PR and The Magician will push any generated updates to the downstream PRs.  Note that if you merge one PR, then another downstream asks for changes which affect the already-merged PR, you will have a messy situation - use caution when many repositories are impacted by your change.
 
-## Individual repo tests
+## Useful `fly` Invocations
 
-The second stage's goal is to confirm that the individual repos still pass tests.  It runs only after the first stage finishes.
+If you want to see what The Magician is going to do to your PR, you'll want to use the `fly` CLI.  You can find documentation about how to use `fly` [here](https://concourse-ci.org/fly.html), but the following invocations may be useful.
 
-### `terraform-test`
+First, you'll need to log in to https://terraform.ci.cloud-graphite.com and download `fly` from there.  You'll need to authenticate `fly` with that target, using `fly login`.  `fly login -t main` will prompt you interactively for the username and password.  [Here are the docs about fly login](https://concourse-ci.org/fly.html#fly-login).
 
-`terraform-test` takes two inputs:
-* The updated `magic-modules` repo, after the robot adds its commit to point the submodules to the new generated version.
-* This CI repository.
+After you're authenticated, the most useful command will be `fly execute`.  [Execute is documented here](https://concourse-ci.org/running-tasks.html).
 
-It then runs `unit-tests/task.yml`, which has no outputs because it makes no changes to any code.  It just runs and succeeds if the unit tests pass, and fails if the unit tests don't pass.  The failure and detailed logs are available in concourse.
+**If you want to see the generated terraform repository**, you should run this command from the root of your repository (after ensuring that the downloaded `fly` is executable and in your PATH):
 
-## `create-prs`
+<!--TODO(ndmckinley) Make these commands resilient to missing files and require no setup.-->
```

Comment:
Fix this TODO?

Reply:
They're certainly a lot more after my next PR, but I want to leave the TODO for now.